Doc/whatsnew/2.7.rst

```diff
@@ -1832,8 +1832,8 @@ https://pypi.python.org/pypi/unittest2.
 
 When used from the command line, the module can automatically discover
 tests.  It's not as fancy as `py.test <http://pytest.org>`__ or
-`nose <http://code.google.com/p/python-nose/>`__, but provides a simple way
-to run tests kept within a set of package directories.  For example,
+`nose <https://code.google.com/archive/p/python-nose>`__, but provides a
```

Comment:
This link seems undated. The new home is more https://nose.readthedocs.io/ no?

Reply:
Seems like you're right about it. The `make linkcheck` command showed me this as a 301. So, I updated it to point to this link.